packages/web-config-server/src/apiV1/dashboards.js

```diff
@@ -88,6 +89,9 @@ export default class extends RouteHandler {
               legacy: item.legacy,
               reportCode: item.report_code,
               ...item.config,
+              isFavourite: userFavouriteDashboardItems
+                ? userFavouriteDashboardItems.includes(item.code)
+                : null,
```

Comment:
@rohan-bes I feel that mapping user favourite dashboard items in frontend is becoming bigger while I am working on part 2..

Adding `isFavourite` here makes thing smooth.

In part 2 I will add a query parameter `favouriteOnly` to allow fetching favourite dashboards only.

Does it look good?

Reply:
Yup, sounds good to me 👍